packages/xrpl/src/Wallet/index.ts

```diff
@@ -197,6 +202,35 @@ class Wallet {
     })
   }
 
+  /**
+   * Derives a wallet from secret numbers.
+   *
+   * @param secretNumbers - A string consisting of 8 times 6 numbers (whitespace delimited) used to derive a wallet.
+   * @param opts - (Optional) Options to derive a Wallet.
+   * @param opts.masterAddress - Include if a Wallet uses a Regular Key Pair. It must be the master address of the account.
```

Comment:
Please add `@param opts.algorithm` + description (Should be a description in the other methods with it)

Reply:
added param + description